docs/developer/contributing/running-kibana-with-apm.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[running-kibana-with-apm]]
+== Running Kibana with the APM Agent Locally
+
+The easiest and recommended way of running Kibana with the APM agent locally is to use the solution provided by the https://github.com/elastic/apm-integration-testing[apm-integration-testing] repo. You’ll need https://www.docker.com/community-edition[Docker] and https://docs.docker.com/compose/install/[Docker Compose] to use the tool.
+
+
+[discrete]
+=== Quick start guide
+
+1. Clone https://github.com/elastic/apm-integration-testing[elastic/apm-integration-testing]
+2. Change into the apm-integration-testing repo:
+** `cd apm-integration-testing`
+3. Run {es} and the APM servers without Kibana using:
+** `./scripts/compose.py start master --no-kibana`
+4. Change into the {kib} repo:
+** `cd ../kibana`
+5. Change the elasticsearch credentials in your `kibana.yml` configuration file to match those needed by elasticsearch and the APM server (see the apm-integration-testing repo's https://github.com/elastic/apm-integration-testing#logging-in[README] for users provided to test different scenarios).
+6. Make sure that the APM agent is active and points to the local APM server by adding the following configuration settings to your `kibana.yml` configuration file:
++
+["source","shell"]
+----------
+elastic.apm.active: true
+elastic.apm.serverUrl: 'http://127.0.0.1:8200'
+# elastic.apm.secretToken: ... <-- might be required in prod/cloud
+# optional metrics to adjust performance 
+elastic.apm.centralConfig: false
+elastic.apm.breakdownMetrics: false
+elastic.apm.transactionSampleRate: 0.1
+elastic.apm.metricsInterval: '120s'
+----------
+
+7. Start Kibana with APM active using:
+** `ELASTIC_APM_ACTIVE=true yarn start`
```

Comment:
nit: do you actually need this env var `ELASTIC_APM_ACTIVE`? I thought it worked with just the `elastic.apm.active: true` config now.

Reply:
I [removed](https://github.com/elastic/kibana/pull/84700/commits/f4d08a0aed5a4d9bc02a7387a4dec6cc292943ea#diff-7e06444f575533e2af08b335d5dbf4a78b03eb69c2ef3f2f9f0ee286442248e4R40) the env var